vendor/github.com/docker/libnetwork/iptables/iptables.go

```diff
@@ -72,11 +72,12 @@ func (e ChainError) Error() string {
 }
 
 func probe() {
-	if out, err := exec.Command("modprobe", "-va", "nf_nat").CombinedOutput(); err != nil {
-		logrus.Warnf("Running modprobe nf_nat failed with message: `%s`, error: %v", strings.TrimSpace(string(out)), err)
+	path, err := exec.LookPath("iptables")
+	if err != nil {
+		return
 	}
-	if out, err := exec.Command("modprobe", "-va", "xt_conntrack").CombinedOutput(); err != nil {
-		logrus.Warnf("Running modprobe xt_conntrack failed with message: `%s`, error: %v", strings.TrimSpace(string(out)), err)
+	if out, err := exec.Command(path, "--wait", "-t", "nat", "-L", "-n").CombinedOutput(); err != nil {
```

Comment:
These files are vendored from the github.com/docker/libnetwork repository; can you open a pull-request in that repository to make those changes?

Reply:
Done, see https://github.com/docker/libnetwork/pull/2351

Reply:
Do I remove the chunk from here? How to update the vendor directory?